testdata/identical_switch_branches.go

```diff
@@ -0,0 +1,77 @@
+package fixtures
+
+func identicalSwitchBranches() {
+	switch a { // MATCH /"switch" with identical branches (lines 6 and 10)/
+	// expected values
+	case 1:
+		foo()
+	case 2:
+		bar()
+	case 3:
+		foo()
+	default:
+		return newError("blah")
+	}
+
+	// MATCH:18 /"switch" with identical branches (lines 20 and 24)/
+	// MATCH:18 /"switch" with identical branches (lines 22 and 26)/
+	switch a {
+	// expected values
+	case 1:
+		foo()
+	case 2:
+		bar()
+	case 3:
+		foo()
+	default:
+		bar()
+	}
```

Comment:
Can you also add an example for

```go
switch a {
  case 1:
  case 2:
  case 3:
   a++
  default:
   a=0
}
```

Where case 1,2 should be grouped


It's the opposite of fallthrought logic